### ISSUE TITLE
Cannot get it to work

### ISSUE THREAD

**Issue by orellabac** [egnyte/gitlabform#242]
**Describe the bug**

I am using a file like this:

config_version: 2
gitlab:
   url: myserver.net
  token: "mytoken"
  api_version: 4

project_and_groups:
   "*":
      files:
         "README.md":
               overwrite: true
               skip_ci: true
               branches:
                   - master
               content:
                  This is my first test!

 
**GitLabForm version**

Output of `gitlabform -V`
2.1.1

But it always says (1/1) Skipping project xxxx/test2 as it has empty effective config

**Comment by gdubicki** [comment]
Hi @orellabac !

Instead of `project_and_groups` you should have `projects_and_groups` - note the plural "projects".

PS It's hard to read the config you provided as it is missing the formatting, which is critical in YAML. In the future please put code and configs into ``` quotes.

**Comment by gdubicki** [comment]
We have just released v2.2.0 that shows a helpful error message if you make a typo in that key name:
```
$ gitlabform -c config-develop.yml gitlabform_test_group -v
🏗  GitLabForm version: 2.2.0 = the latest stable 😊
Reading config from file: config-develop.yml
Connected to GitLab version: (...)
Error: Configuration has to contain non-empty 'projects_and_groups' key.
```
As I hope that this was your only problem, @orellabac and as you are not responding, I am closing this issue. But please reopen if needed!

**Comment by orellabac** [comment]
Thanks !! Yes my issues was just that
